### PR TITLE
California exemptions

### LINKED ISSUE #1066: California renter's credit
https://casetext.com/statute/california-codes/california-revenue-and-taxation-code/division-2-other-taxes/part-10-personal-income-tax/chapter-2-imposition-of-tax/section-170535-credit-for-qualified-renter

### PR DESCRIPTION
**Pull request by josh-marsh**
fixes #1066 
Co-authored-by: Max Ghenis <mghenis@gmail.com>

Thanks for contributing! Please remove any top-level sections that do not apply to your changes.

- [ ] `make format && make documentation` has been run.

# New variable

- [ ] Label field added
- [ ] Documentation field added
- [ ] Unit field added
- [ ] Default value field added if relevant
- [ ] Variable name follows conventions
- [ ] Unit test(s) added
- [ ] Integration test(s) added if relevant
- [ ] Issues this PR fixes linked

## What's changed

Description of the changes here.

# Bug fix

- [ ] Regression test added
- [ ] Regression test passing

## What this fixes and how it's fixed

Description of how this fix works goes here. Link any issues this PR fixes.

### CODE REVIEW

File: openfisca_us/variables/gov/states/ca/tax/income/exemptions/ca_exemptions.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        aged_blind_count = tax_unit("aged_blind_count", period)
+        blind_senior_exemption = aged_blind_count * (
+            p.amount - exemption_reduction
+        )
+        blind_senior_exemption = max_(0, blind_senior_exemption)
+
+        # Dependent exemptions
+        person = tax_unit.members
+        is_dependent = person("is_tax_unit_dependent", period)
+        num_dependent = tax_unit.sum(is_dependent, period)
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/gov/states/ca/tax/income/exemptions/ca_exemptions.py:39]: This is what's causing the `2021 is not a valid role` error - the second argument should be a role if a country implements them (for example, "sum employment income for all adults in the household"). We don't use this feature in the US system, so we can just shorten this to:

```suggestion
        num_dependent = tax_unit.sum(is_dependent)
```

File: openfisca_us/tests/policy/baseline/gov/states/ca/tax/income/exemptions/ca_exemptions.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  input:
+    people:
+      parent: 
+        age: 25
+      spouse: 
+        age: 25
+      child: 
+        age: 4
+        is_tax_unit_dependent: True 
+      tax_unit:
```
Comment (nikhilwoodruff) [inline openfisca_us/tests/policy/baseline/gov/states/ca/tax/income/exemptions/ca_exemptions.yaml:73]: `tax_unit` is defined correctly here, but it should be under the parent `tax_units`, not `people`.

**Review by nikhilwoodruff — COMMENTED** [review]
That error is pretty cryptic! Found the source, LMK if anything doesn't make sense here. Thanks!

**Comment by MaxGhenis** [comment]
Thanks @josh-marsh, and thanks @PavelMakarchuk for getting this across the finish line! Merging.
